packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.2+2
+
+* Add `analysis_options.yaml` to the package, so we can ignore `undefined_prefixed_name` errors (temporarily).
```

Comment:
Please add an issue link, TODO: remove ...

Reply:
oh. Just saw this. We can also add the TODO in the anlaysis_options file, @ditman Your call.

Reply:
actually, I don't think we should add TODO in the CHANGELOG. let's add to the `analysis_options` file?

Reply:
I don't think it makes sense to put a TODO in the changelog, but an issue link here would be fine.